cmake/FDBInstall.cmake

```diff
@@ -0,0 +1,258 @@
+function(fdb_install_packages)
+  set(FDB_INSTALL_PACKAGES ${ARGV} PARENT_SCOPE)
+endfunction()
+
+function(fdb_install_dirs)
+  set(FDB_INSTALL_DIRS ${ARGV} PARENT_SCOPE)
+endfunction()
+
+function(install_symlink_impl)
+  if (NOT WIN32)
+    set(options "")
+    set(one_value_options TO DESTINATION)
+    set(multi_value_options COMPONENTS)
+    cmake_parse_arguments(SYM "${options}" "${one_value_options}" "${multi_value_options}" "${ARGN}")
+
+    file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/symlinks)
+    get_filename_component(fname ${SYM_DESTINATION} NAME)
+    get_filename_component(dest_dir ${SYM_DESTINATION} DIRECTORY)
+    set(sl ${CMAKE_CURRENT_BINARY_DIR}/symlinks/${fname})
+    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${SYM_TO} ${sl})
+    foreach(component IN LISTS SYM_COMPONENTS)
+      install(FILES ${sl} DESTINATION ${dest_dir} COMPONENT ${component})
+    endforeach()
+  endif()
+endfunction()
+
+function(install_symlink)
+  if(NOT WIN32 AND NOT OPEN_FOR_IDE)
+    set(options "")
+    set(one_value_options COMPONENT LINK_DIR FILE_DIR LINK_NAME FILE_NAME)
+    set(multi_value_options "")
+    cmake_parse_arguments(IN "${options}" "${one_value_options}" "${multi_value_options}" "${ARGN}")
+
+    set(rel_path "")
+    string(REGEX MATCHALL "\\/" slashes "${IN_LINK_NAME}")
+    foreach(ignored IN LISTS slashes)
+      set(rel_path "../${rel_path}")
+    endforeach()
+    if("${IN_FILE_DIR}" MATCHES "bin")
+      if("${IN_LINK_DIR}" MATCHES "lib")
+        install_symlink_impl(
+          TO "../${rel_path}bin/${IN_FILE_NAME}"
+          DESTINATION "lib/${IN_LINK_NAME}"
+          COMPONENTS "${IN_COMPONENT}-tgz")
+        install_symlink_impl(
+          TO "../${rel_path}bin/${IN_FILE_NAME}"
+          DESTINATION "usr/lib64/${IN_LINK_NAME}"
+          COMPONENTS "${IN_COMPONENT}-el6"
```

Comment:
Not sure if you want to remove the 'el6' component.

Reply:
see #3973 - we discussed this in our meeting two weeks ago and decided to drop el6 support as el6 is end of life for a while already.

Reply:
ooh, you're right Andrew, somehow I missed that...